model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -595,6 +606,9 @@ object WorkspaceSettingConfig {
   case class GcpBucketLifecycleRule(action: GcpBucketLifecycleAction, conditions: GcpBucketLifecycleCondition)
   case class GcpBucketLifecycleAction(actionType: String)
   case class GcpBucketLifecycleCondition(matchesPrefix: Option[Set[String]], age: Option[Days])
+
+  type Seconds = Long
```

Comment:
It's a little odd to have a broad type alias like this in a narrow class. That said, I like the expressiveness.